content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
-will be marked :guilabel:`Ready` only after the transfer to quality has been marked
-:guilabel:`Done`.
+The receipt can be confirmed from the original purchase order form, or it can be accessed by
+navigating to the :menuselection:`Inventory app`, and locating the :guilabel:`Receipts` task card.
 
-The receipt can also be found in the :menuselection:`Inventory` application. In the
-:guilabel:`Overview` dashboard, click the :guilabel:`1 To Process` smart button in the
-:guilabel:`Receipts` kanban card.
-
-.. image:: receipts_three_steps/three-step-receive-kanban.png
-   :align: center
-   :alt: One Receipt ready to process in the Inventory Overview kanban view.
+Click the :guilabel:`# To Process` button to reveal all incoming receipts to process. Click the
+receipt associated with the previous purchase order.
 
-Click on the receipt associated with the purchase order, then click :guilabel:`Validate` to complete
-the receipt and move the product to the :guilabel:`Input Location`.
+Click :guilabel:`Validate` to validate the receipt, and move the product to the destination
+location, :guilabel:`WH/Input`.
 
-.. image:: receipts_three_steps/validate-three-step-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-receipt-form.png
    :align: center
-   :alt: Validate the receipt by clicking Validate, and the product will be transferred to the
-         WH/Quality location.
+   :alt: Receipt operation for product being moved to WH/Input location.
 
 Process a transfer to Quality Control
 -------------------------------------
 
-Once the product is in the :guilabel:`Input Location`, the internal transfer is ready to move the
-product to :guilabel:`Quality Control`. In the :menuselection:`Inventory` :guilabel:`Overview`
-dashboard, click the :guilabel:`1 To Process` smart button in the :guilabel:`Internal Transfers`
-kanban card.
+Once the receipt has been validated, an internal transfer operation to move the product to quality
+control is ready to process.
 
-.. image:: receipts_three_steps/three-step-quality-transfer.png
-   :align: center
-   :alt: One Internal Transfer ready to process in the Inventory Overview kanban view.
+Click :guilabel:`Inventory Overview` in the breadcrumbs to navigate back to the dashboard, and
+locate the :guilabel:`Internal Transfers` task card.
+
+Click the :guilabel:`# To Process` button to reveal all internal transfers to process. Then, click
+the internal transfer associated with the validated receipt.
 
-Click on the :guilabel:`Transfer` associated with the purchase order, then click
-:guilabel:`Validate` to complete the transfer and move the product to the :guilabel:`Quality
-Control` location. Once the transfer is validated, the product is ready for the quality inspection,
-but is not available for manufacturing or delivery orders.
+Click :guilabel:`Validate` to complete the transfer, and move the product from :guilabel:`WH/Input`
+to :guilabel:`WH/Quality Control`.
```

Comment:
All three of these paragraphs begin with the word **Click**.
Consider using more varied or transitional language, please.

Reply:
Changed some words in the second and third paragraphs per the instructions from this suggestion 🙂 Suggestion implemented ✅